src/django/api/reports.py

```diff
@@ -4,15 +4,35 @@
 from urllib.parse import quote
 
 from django.db import connection
-
+from api.models import Facility
 
 _root = os.path.abspath(os.path.dirname(__file__))
 
 
+def monthly_promoted_name_and_address():
+    data = dict()
+    fs = Facility.objects.all()
+    for f in fs:
+        hs = f.history.exclude(created_from_id__isnull=True)
+        for h in hs:
+            prev = h.prev_record
+            if prev is not None and prev.created_from_id is not None \
+                    and h.created_from_id is not prev.created_from_id:
```

Comment:
I think we should prefer `!=` over `is not` when compare numeric values. I recall a previous usage of `is not` with numbers resulting in a bug (https://realpython.com/python-is-identity-vs-equality/).

Reply:
Thanks! I need to keep that difference in mind. 
Updated in 44a9a1f